lib/express_validator.js

```diff
@@ -124,7 +125,12 @@ var expressValidator = function(options) {
     var locations = ['body', 'params', 'query'];
 
     req._validationErrors = [];
-    req.validationErrors = function(mapped) {
+    req._asyncValidationErrors = [];
+    req.validationErrors = function(mapped, promisesResolved) {
+      if (!promisesResolved && req._asyncValidationErrors.length > 0) {
+        console.warn('WARNING: You have asynchronus validators but you have not used asyncValidateErrors to check for errors.');
```

Comment:
Also, not sure what the correct answer is here... but should we only do a "warn" or should we also throw an Error? What are the consequences of using asynchronous validators without `asyncValidateErrors`?